airbyte-integrations/connectors/source-facebook-marketing/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1",
+    "airbyte-cdk>=0.2.3",
```

Comment:
How come we are adding the patch version back? Looking at https://peps.python.org/pep-0440/#compatible-release , if we use format `>=0.2.3` we'll get pinned to patch versions like `==0.2.*`

Reply:
Using patch granularity because the fix was introduced in `0.2.3`. It might not matter in practice since I'd expect it to always use the latest `0.2.*`

Reply:
i think my concern is that if we bump the CDK for new features like 0.3.1 for ex, these two connectors will stop receiving updates